src/Arcus.Security.Tests.Unit/Core/Extensions/IHostBuilderExtensionsTests.cs

```diff
@@ -471,7 +471,7 @@ public async Task ConfigureSecretStore_WithDefaultAuditing_DoesntLogsSecurityEve
             IHost host = builder.Build();
             var secretProvider = host.Services.GetRequiredService<ISecretProvider>();
             await secretProvider.GetRawSecretAsync(secretName);
-#if NET6_0
+#if NET6_0 || NET8_0
```

Comment:
We only support .NET 6 & .NET 8 ?

Reply:
This is the test project, this makes sure that we use the correct assertion when we run the tests. But since we have removed .NET Core from this project, we can probably remove this.

Reply:
Since the define is still here, I believe we cannot remove this (yet) ?
Is this because we're using another (Arcus) package and the output we're checking is generated by that package ?

Reply:
Removed this, forgotten. 😅 Thx for keeping a sharp eye.

Reply:
Removed the directives here: https://github.com/arcus-azure/arcus.security/pull/421/commits/d24b66901b7efe25e896f007b5b6a323722a58c4